packages/iris-grid/src/GotoRow.tsx

```diff
@@ -281,6 +286,7 @@ function GotoRow({
             <div className="goto-row-input">
               <input
```

Comment:
Forgot to respond in Slack to this question, but you _can_ use `idi` if you generate the id using `shortid` or something similar. See `DropdownFilter.tsx` for an example of doing that. It's probably best to have the label linked with the input correctly as you had in Slack.

Reply:
I was thinking of applying this label change but for the "Go to value" row there's only one label and 2 inputs (column selector and value to search). In the scenario that the column is strings there actually ends up being 3 inputs. Should I just be linking the label with one of those inputs? If so, which one should it be?

Reply:
For one label on multiple inputs, it would be more correct to use a [fieldset](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) and [legend](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/legend) instead of a label. Something like:
```
<fieldset>
  <legend>Go to value</legend>
  ...
  <select aria-label="Filter type" ... />
  <input aria-label="Value input" .... />
</fieldset>
```
Then you should be able to do `screen.getByRole('group', { name: 'Go to value' })` to get the `fieldset`, or get the individual inputs by `screen.getByLabelText("Filter type")`
Docs for [accessibility here](https://www.w3.org/TR/html-aam-1.0/#fieldset-element-accessible-name-computation).